test/response-mook-1-notif.json

```diff
@@ -0,0 +1,62 @@
+{
+    "notifications": [
```

Comment:
Name of file should be more explicit `response-mook-1` not working.

Example of good name -> goodResponseWith2NotificationsMock